### ISSUE TITLE
KeyError: TypeID in execuution output (500 Internal Server Error)

### ISSUE THREAD

**Issue by eudoxos** [mupif/mupifDB#8]
Log when accessing http://172.23.1.1:5555/workflowexecutions/620da8c3d3688fd7e40a1f29/outputs :
```python
15:05:28 [707|Thread-994] INFO:_internal.py:225 172.23.1.18 - - [14/Mar/2022 15:05:28] "GET /workflowexecutions/620da8c3d3688fd7e40a1f29/outputs HTTP/1.1" 500 -
15:05:47 [707|Thread-995] ERROR:app.py:1457 Exception on /workflowexecutions/620da8c3d3688fd7e40a1f29/outputs [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/dist-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/var/lib/mupif/mupifDB/webapi/index.py", line 612, in getExecutionOutputs
    form += '<td>' + str(i['TypeID']).replace('mupif.DataID.', '') + '</td>'
KeyError: 'TypeID'
15:05:47 [707|Thread-995] INFO:_internal.py:225 172.23.1.18 - - [14/Mar/2022 15:05:47] "GET /workflowexecutions/620da8c3d3688fd7e40a1f29/outputs HTTP/1.1" 500 -
```

**Comment by eudoxos** [comment]
https://github.com/mupif/mupifDB/blob/c62e07545b341b8548159ef9b521d1d1b1f243a2/webapi/index.py#L612

I'd suggest perhaps `i['TypeID']` → `i.get('TypeID','[unknown]')`

**Comment by eudoxos** [comment]
Fixed in da470d898ac38ebd5d6c87662d141894a6331035 (cc @bpatzak)
